### ISSUE TITLE
Allow BioProcess as controller?

### ISSUE THREAD

**Issue by bgyori** [clulab/reach#682]
Currently, in most instances, controllers of modifications and activity/amount regulations are constrained to being a BioChemicalEntity, which is reasonable if the assumption is that we are representing only direct physical interactions. However, for indirect regulations in general, BioProcess-es should be allowed to be controllers. For instance "DNA damage leads to the phosphorylation of H2AX" is a perfectly valid relationship in which "DNA damage", a BioProcess (indirectly) controls the phosphorylation of H2AX - currently these are not extracted. It probably requires some empirical evaluation on a sample corpus to determine if making this change would be overall useful or not (it may lead to noisy extractions).

**Comment by MihaiSurdeanu** [comment]
Good point. I believe BioProcess can serve as Controlled but not as Controller (I'll have to double check that). I agree we should allow it as Controller, in theory. A small eval would be great!
